packages/manager/src/features/VPC/VPCLanding/VPCDeleteDialog.tsx

```
@@ -0,0 +1,47 @@
+import { useSnackbar } from 'notistack';
+import * as React from 'react';
+
+import { TypeToConfirmDialog } from 'src/components/TypeToConfirmDialog/TypeToConfirmDialog';
+import { useDeleteVPCMutation } from 'src/queries/vpcs';
+
+interface Props {
+  id: number;
+  label: string;
+  onClose: () => void;
+  open: boolean;
+}
+
+export const VPCDeleteDialog = (props: Props) => {
+  const { id, label, onClose, open } = props;
+  const { enqueueSnackbar } = useSnackbar();
+  const { error, isLoading, mutateAsync: deleteVPC } = useDeleteVPCMutation(id);
+
+  const onDeleteVPC = () => {
+    deleteVPC()
+      .then(() => {
+        enqueueSnackbar('VPC deleted successfully.', {
+          variant: 'success',
+        });
+        onClose();
+      })
+      .catch();
```

We could use async/await here and empty `catch` block may leads to silent errors.

---

I think the error from the mutation query should cover this. Does that sound right? @bnussman-akamai